benches/comparative.rs

```diff
@@ -0,0 +1,86 @@
+#[macro_use]
+extern crate criterion;
+
+#[macro_use]
+extern crate yoga;
+
+extern crate stretch;
+
+use criterion::Criterion;
+use criterion::Fun;
+use yoga::prelude::*;
+use yoga::Node;
+use yoga::StyleUnit::{Auto, UndefinedValue};
+
+fn yoga(n: u64) {
+    let mut node = Node::new();
+
+    let mut child = Node::new();
+    let mut other_child = Node::new();
+
+    node.insert_child(&mut child, 0);
+    node.insert_child(&mut other_child, 1);
+
+    style!(node,
+        Margin(10 pt),
+        MarginLeft(Auto),
+        PaddingHorizontal(4 pt),
+        Left(16 %),
+        Bottom(UndefinedValue)
+    );
+
+    let child_styles = make_styles!(
+        Width((n as f32) pt),
+        Height(32 pt),
+        FlexGrow(1.0),
+        Margin(Auto)
+    );
+
+    child.apply_styles(&child_styles);
+    other_child.apply_styles(&child_styles);
+
+    node.calculate_layout(512.0, 512.0, yoga::Direction::LTR);
```

Comment:
Stretch currently does not have a way to pass in parent dimensions like Yoga has, this could easily be added later on but for now stretch assumes parent size of undefined/undefined.

```
node.calculate_layout(UndefinedValue, UndefinedValue, yoga::Direction::LTR);
```